spec/integration/token_authentication_spec.rb

```diff
@@ -14,14 +14,14 @@
   end
 
   context 'with correct token query params' do
-    it 'returns the root presenter' do
+    xit 'returns the root presenter' do
       client.params['token'] = 'replace-me'
       expect(client._links.self._url).to eq client_url
     end
   end
 
   context 'with correct Http-Authorization headers' do
-    it 'returns the root presenter' do
+    xit 'returns the root presenter' do
```

Comment:
Temporarily pending while I take a look into how we may be able to set these up differently - https://github.com/artsy/gris/issues/65